ros/finalarm_description/urdf/finalarm_description.urdf.xacro

```diff
@@ -0,0 +1,504 @@
+  </joint>
+  <link name="link_9">
+    <inertial>
+      <origin
+        xyz="0.0330748692850015 -0.0156609082500656 -0.000838888003525384"
+        rpy="0 0 0"
+      />
+      <mass value="0.0633522380056329" />
+      <inertia
+        ixx="0.00002229791"
+        ixy="-0.00001713184"
+        ixz="0.00000102093"
+        iyy="0.00008811855"
+        iyz="-0.00000048759"
+        izz="0.00009857025"
+      />
+    </inertial>
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0" />
+      <geometry>
+        <mesh filename="package://finalarm_description/meshes/link_9.STL" />
+      </geometry>
+      <material name="">
+        <color rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
+      </material>
+    </visual>
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0" />
+      <geometry>
+        <mesh filename="package://finalarm_description/meshes/link_9.STL" />
+      </geometry>
+    </collision>
+  </link>
+  <joint name="joint_8" type="revolute">
+    <origin xyz="-0.0174 0 -0.057005" rpy="1.5708 1.5708 0" />
+    <parent link="link_8" />
+    <child link="link_9" />
+    <axis xyz="0 0 1" />
+    <limit lower="-3.14" upper="3.14" effort="6" velocity="2.0" />
+    <dynamics damping="0.5" />
+  </joint>
+  <link name="link_10">
+    <inertial>
+      <origin
+        xyz="0.0338709492936103 0.0160630015014153 -0.000269069197496835"
+        rpy="0 0 0"
+      />
+      <mass value="0.0618364235211619" />
+      <inertia
+        ixx="0.00002129955"
+        ixy="0.00001626282"
+        ixz="-0.00000001937"
+        iyy="0.00008586701"
+        iyz="-0.00000001339"
+        izz="0.00009640957"
+      />
+    </inertial>
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0" />
+      <geometry>
+        <mesh filename="package://finalarm_description/meshes/link_10.STL" />
+      </geometry>
+      <material name="">
+        <color rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
+      </material>
+    </visual>
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0" />
+      <geometry>
+        <mesh filename="package://finalarm_description/meshes/link_10.STL" />
+      </geometry>
+    </collision>
+  </link>
+  <joint name="joint_9" type="revolute">
+    <origin xyz="0.0186 0 -0.056505" rpy="1.5708 1.5708 0" />
+    <parent link="link_8" />
+    <child link="link_10" />
+    <axis xyz="0 0 1" />
+    <limit lower="-3.14" upper="3.14" effort="6" velocity="2.0" />
+    <dynamics damping="0.5" />
+  </joint>
+  <link name="link_eef" />
+  <joint name="joint_eef" type="fixed">
+    <origin xyz="0 0 -0.15225" rpy="3.1416 0 3.1416" />
+    <parent link="link_8" />
+    <child link="link_eef" />
+    <axis xyz="0 0 0" />
+  </joint>
+
+  <xacro:if value="$(arg use_gazebo)">
+    <xacro:macro name="static_no_collide" params="reference">
+      <gazebo reference="${reference}">
+        <turnGravityOff>False</turnGravityOff>
+        <selfCollide>False</selfCollide>
+      </gazebo>
+    </xacro:macro>
```

Comment:
The macros ended up being pretty nice here, though, though I wanted to use loops of some kind.